controllers/suite_test.go

```diff
@@ -77,6 +79,17 @@ const (
 )
 
 func TestAPIs(t *testing.T) {
+
+	// check for which deployment type we need to run the tests for
+	envVar := "DEPLOYMENT_TYPE"
```

Comment:
Why not a const in the consts section above? 
Also, this is the name of the env var so can we be more descriptive and call this `deploymentTypeEnvVarName`?